packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -339,7 +339,7 @@ export const mutation = mutationType({
       resolve (_, args, ctx) {
         ctx.actions.auth.resetAuthState()
 
-        return ctx.appData
+        return {}
```

Comment:
We aren't reading this data from the mutation, so I removed it to align with some of the other mutations.